scripts/plotting/make_postfit_vgen.py

```diff
@@ -65,7 +67,7 @@ def quadrature_sum_hist(hists, is_down):
 
 def load_hist(filename, fittype="postfit", helicity=False):
     fitresult = combinetf2_input.get_fitresult(filename)
-    obs = {args.obs, "helicity"} if helicity else {args.obs}
+    obs = {args.obs, "helicity", "chargeVgen"} if helicity else {args.obs}
```

Comment:
Do I understand correctly that from now on the script will only deal with cross section by charge? So far we were plotting charge integrated cross sections, but with this change it seems the charge axis will have to be present within the inputs (maybe it always does, but so far it was probably not the case)